nginx.conf

```diff
@@ -0,0 +1,42 @@
+worker_processes  auto;
+error_log /dev/stdout info;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log /dev/stdout;
+
+    sendfile        on;
+    
+    keepalive_timeout  65;
+
+    gzip  on;
+
+    server {
+        listen       80;
+        listen 443 default_server ssl;
+
+        server_name         github.com;
+        ssl_certificate /etc/ssl/certs/github.com.crt;
+        ssl_certificate_key /etc/ssl/private/github.com.key;
+
+        location /istio/istio/releases/download/ {
+            alias   /opt/istio-releases/;
+        }
+
+        error_page   500 502 503 504  /50x.html;
+        location /50x.html {
+            root   /usr/share/nginx/html;
```

Comment:
This looks like there was an intention to align properties and values. Needs to be reformatted.

Reply:
downloaded an nginx formatter and used that to format the file